tutorials/scripting/c_sharp/c_sharp_signals.rst

```diff
@@ -0,0 +1,147 @@
+.. _doc_c_sharp_signals:
+
+C# Signals
+==========
+
+For a detailed explanation of signals in general, see the :ref:`doc_signals` section in the step
+by step tutorial.
+
+If it is still possible to use signals through the ``Connect``/``Disconnect`` API, C# gives us
+a more idiomatic way into the observer pattern.
+
+Signals as C# events
```

Comment:
Do we want to add a note that `Invoke` can't be used to raise events? Or is that sufficiently clear with the _Signal emission_ section below?

Reply:
I almost wrote something about it, so if you thought about it too, it's probably necessary.